magento/magento2
# Backend: Global select box arrow issue. when user select any option wrong arrow icon displayed.

### Description (*)

Backend: Global select box arrow issue. when user select any option wrong arrow icon displayed.
### Fixed Issues (if relevant)

1. This is direct pr as per suggest magento reviewer.
2. Backend: Global select box arrow issue. when user select any option wrong arrow icon displayed.

### Manual testing scenarios (*)

1. Login into admin panel > Content > Configuration > Select any theme > Click on applied theme > choose any other theme. Arrow direction is wrong.

    Note: This change for global select box.
2. Actual Result.
![actual-result-1](https://user-images.githubusercontent.com/43638601/51072622-ca303b80-1689-11e9-8f94-49b7edadcc69.png)

3. Expected Result.
![expected-result](https://user-images.githubusercontent.com/43638601/51072627-d9af8480-1689-11e9-980c-2c01c1d3f60e.png)


### Contribution checklist (*)
